feat(#2136): add textarea example for long question

bdfranck opened the pull request:
This PR adds a  `textarea` example for asking a question with a long response.

<img width="858" alt="image" src="https://github.com/user-attachments/assets/7e927b35-8630-4def-81aa-71e77fbe3068" />

twjeffery reviewed (changes requested):
The example looks good! A couple small things to add/change:
- update the number of examples from 1 to 2

<img width="317" alt="image" src="https://github.com/user-attachments/assets/aab3dcd0-a73f-40fe-a643-db3a8f612acb" />

- build out, publish, and link to the example in the pattern library in Figma, and add the Figma link to the example heading (let me know if you have any questions)

bdfranck commented:
> The example looks good! A couple small things to add/change:
> 
> * update the number of examples from 1 to 2
> 
> <img alt="image" width="317" src="https://private-user-images.githubusercontent.com/82968683/441912324-aab3dcd0-a73f-40fe-a643-db3a8f612acb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY4MDIyMDEsIm5iZiI6MTc0NjgwMTkwMSwicGF0aCI6Ii84Mjk2ODY4My80NDE5MTIzMjQtYWFiM2RjZDAtYTczZi00MGZlLWE2NDMtZGIzYThmNjEyYWNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA5VDE0NDUwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmY2QwMzNjNTllZGIwNWYxNjVkODBlYjA3MzRiMjEwYzY4MTE5ZTA4YjIzM2E5NDJmZGVkYTYyYTM4ZWQwYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GTBqjNzQbyCEpVL_Gv5DCpMV07ePsc6FHAaNJhHR5e0">
> * build out, publish, and link to the example in the pattern library in Figma, and add the Figma link to the example heading (let me know if you have any questions)

Good catch! I've amended my commit wit the following changes:
 
- Changed the tab number from 1 to 2
- Added the link to the Figma example

bdfranck commented:
@twjeffery Thanks for catching those! I've amended my commit with an updated number of examples and included the Figma link.

bdfranck commented:
I also realized that "long-answer question" should include a hyphen. I've added one.

<img width="357" alt="image" src="https://github.com/user-attachments/assets/e8e37bac-85f7-4f27-9624-e41bcbb2bb06" />

twjeffery reviewed (approved):
Looks great now, thanks for making those changes. 

### One more consideration:
When filling in the text area it's not a lot of working area to show 500 words, it has to scroll fairly quickly and the input stays the same height, limiting the view. It would be nice to add a grab bar on the edge of the text area to drag it taller. 

In the meantime for this example, maybe we should start it as 5 or 6 lines tall just so it has a bit more room?

<img width="830" alt="image" src="https://github.com/user-attachments/assets/482a35ed-64c9-4b8c-acad-47480c061df8" />

bdfranck commented:
@twjeffery Thanks! I've increased the `textarea` height to 6 rows. 👍
